oak_restricted_kernel/src/interrupts.rs

```diff
@@ -164,13 +164,164 @@ mutable_interrupt_handler_with_error_code!(
     }
 );
 
+extern "x86-interrupt" fn divide_error_handler(stack_frame: InterruptStackFrame) {
+    error!("KERNEL PANIC: DIVIDE BY ZERO!");
+    error!(
+        "Instruction pointer: {:#016x}",
+        stack_frame.deref().instruction_pointer.as_u64()
+    );
+    shutdown::shutdown();
+}
+
+extern "x86-interrupt" fn nmi_handler(stack_frame: InterruptStackFrame) {
```

Comment:
If you think this list is something that we will keep long term, it may be interesting to use a macro to reduce boilerplate.

Reply:
(in a future PR, not here)